### PR TITLE
Acs Poverty Providers

### PR DESCRIPTION
**Pull request by Sethvg**

### CODE REVIEW

File: frontend/src/data/variables/AcsPovertyProvider.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      below_poverty_line: (series: ISeries) => series.sum(),
+    });
+
+    return df;
+  }
+
+  allowsBreakdowns(breakdowns: Breakdowns): boolean {
+    return (
+      breakdowns.hasExactlyOneDemographic() &&
+      !breakdowns.hasOnlyAge() &&
```
Comment (vanshkumar) [inline frontend/src/data/variables/AcsPovertyProvider.ts:114]: Is there a specific reason why age breakdown is not supported here?

**Review by vanshkumar — COMMENTED** [review]
Generally LGTM, have you tested this in the FE to make sure it works?

File: frontend/src/data/variables/AcsPovertyProvider.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
+import { Breakdowns } from "../query/Breakdowns";
+import { per100k } from "../utils/datasetutils";
+import { USA_FIPS, USA_DISPLAY_NAME } from "../utils/Fips";
+import VariableProvider from "./VariableProvider";
+import { MetricQuery, MetricQueryResponse } from "../query/MetricQuery";
+import { getDataManager } from "../../utils/globals";
+import { TOTAL } from "../utils/Constants";
+import { IDataFrame, ISeries } from "data-forge";
+
+const ABOVE_POVERTY_COL = "above_poverty_line";
```
[the remark this answers is not in the document]
Comment (Sethvg) [reply frontend/src/data/variables/AcsPovertyProvider.ts]: Fixed

**Review by vanshkumar — COMMENTED** [review]
Can you fix the build issue so I can see the deploy preview? Otherwise looks good to me

File: frontend/src/utils/MadLibs.ts [hunk 3]
```diff
@@ -70,6 +71,7 @@ const DROPDOWN_VAR: Record<DropdownVarId, string> = {
   //  asthma: "[coming soon] asthma",
   copd: "COPD",
   health_insurance: "Health Insurance",
+  poverty: "poverty",
```
Comment (vanshkumar) [inline frontend/src/utils/MadLibs.ts:74]: Can you capitalize this to match the other data?

**Review by vanshkumar — APPROVED** [review]
Can you also fill out FakeDatasetMetadata for this dataset so the sources / last updated show up properly? This file: https://github.com/SatcherInstitute/health-equity-tracker/blob/2c4f8136e10a0ee96a6a4aba67a1359686004ef9/frontend/src/data/config/FakeDatasetMetadata.ts

**Pull request merged**